pkg/asset/agent/image/agentartifacts.go

```diff
@@ -64,7 +65,18 @@ func (a *AgentArtifacts) Generate(dependencies asset.Parents) error {
 	a.RendezvousIP = ignition.RendezvousIP
 	a.IgnitionByte = ignitionByte
 	a.ISOPath = baseIso.File.Filename
-	a.Kargs = kargs.KernelCmdLine()
+
+	platform := agentManifests.AgentClusterInstall.Spec.PlatformType
+	var platformName, consoleArgs string
+	if agentManifests.AgentClusterInstall.Spec.ExternalPlatformSpec != nil {
+		platformName = agentManifests.AgentClusterInstall.Spec.ExternalPlatformSpec.PlatformName
+	}
+	if platform == hiveext.ExternalPlatformType && platformName == "oci" {
+		consoleArgs = " console=ttyS0"
+	}
```

Comment:
I'd like this code to be inside the `KernelCmdLine()` function. This is possible by making ACI a dependency of Kargs.